plugins/inputs/cloud_pubsub/README.md

```diff
@@ -101,6 +101,15 @@ See the [CONFIGURATION.md][CONFIGURATION.md] for more details.
   ## PubSub message data before parsing. Many GCP services that
   ## output JSON to Google PubSub base64-encode the JSON payload.
   # base64_data = false
+
+  ## Content encoding for message payloads, can be set to "gzip" or
+  ## "identity" to apply no encoding.
+  # content_encoding = "identity"
+
+  ## Maximum size of decoded message.
+  ## Acceptable units are B, KiB, KB, MiB, MB...
+  ## Without quotes and units, interpreted as size in bytes.
+  # max_decompression_size = "500MB"
```

Comment:
I don't think the lines about units or quotes are needed. Instead can you explain to the user why they would want or need to change this value?

Reply:
Since I copied this descriptions over from other plugins. Should I update those plugin's descriptions? Or just keep it scoped to cloud_pubsub?

Reply:
ha! I didn't know that. Let's continue with the change only for this plugin please.

Reply:
Seems like when I tried to update the description it reverts back to the old one when `make docs` is run.

Here is the description I tried to update to
```
  ## Sets the maximum allowed size, in bytes, for a message 
  ## payload when it's decompressed. You can increase it for larger payloads 
  ## or reduce it to limit possible resource consumption. For identity encoding,
  ## this simply limits the size of each message payload. 
  ## Acceptable units are B, KiB, KB, MiB, MB...
  # max_decompression_size = "500MB"
```

Reply:
Ah right - I always comment on the README since it shows up first, but you have to edit the sample.conf. When you run `make docs` it will automatically update the README for you.

Reply:
Considering the way the `internal` decoder uses the max_decompression_size [here](https://github.com/dayvar14/telegraf/blob/0636faf67ca109b2c46a683fc5178f939a69f541/internal/content_coding.go#L288-L294), this description seems correct, but can this be considered possibly breaking if a customer currently sends message payloads greater than the default size specified by [DefaultMaxCompressionSize](https://github.com/dayvar14/telegraf/blob/0636faf67ca109b2c46a683fc5178f939a69f541/internal/content_coding.go#L13)?

Reply:
Is this behavior expected from the internal decoder? What is the reason for an identity decoder with a max_decompression_size limit?

Reply:
It was a recent change actually to protect against decompression bomb, via gosec's G110 rule

Reply:
Feels odd that a 'identity' content encoder requires this field, as it simply ends up being a size limit rather than a decompression size limit. Regardless, I updated this to prevent introducing a breaking change.